runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CrypticClue.java

```diff
@@ -61,129 +62,248 @@ public class CrypticClue extends ClueScroll implements NpcClueScroll, ObjectClue
 {
 	static final List<CrypticClue> CLUES = ImmutableList.of(
 		CrypticClue.builder()
+			.itemId(ItemID.CLUE_SCROLL_ELITE_12113)
 			.text("Show this to Sherlock.")
 			.location(new WorldPoint(2733, 3415, 0))
 			.npc("Sherlock")
 			.solution("Sherlock is located to the east of the Sorcerer's tower in Seers' Village.")
 			.build(),
 		CrypticClue.builder()
+			.itemId(ItemID.CLUE_SCROLL_ELITE_12114)
+			.text("Show this to Sherlock.")
+			.location(new WorldPoint(2733, 3415, 0))
+			.npc("Sherlock")
+			.solution("Sherlock is located to the east of the Sorcerer's tower in Seers' Village.")
+			.build(),
```

Comment:
For ease of use here, perhaps associating a set of IDs instead of duplicating the clue many times over is the better approach.

Reply:
Yeah I can look at having itemId accept a list of IDs, since 12113-12127 are all sherlock clues.

Reply:
itemId in CrypticClue now expects Set.of